src/main/java/seedu/duke/storage/Loader.java

```diff
@@ -0,0 +1,213 @@
+package seedu.duke.storage;
+
+import seedu.duke.common.Constants;
```

Comment:
Consider replacing this line with:

import static seedu.duke.common.Constants.(constant name)

so you don't have to repeatedly use Constants.(constant name)? I think if you are using most of the constants in that class, it is acceptable to use *.